modules/sdk-coin-near/src/near.ts

```diff
@@ -495,7 +510,11 @@ export class Near extends BaseCoin {
       },
     });
     if (response.status !== 200) {
-      throw new Error('Account not found');
+      throw new Error('Failed to query account information');
+    }
+    const errorCause = response.body.error?.cause.name;
```

Comment:
Will `cause` always have a `name` property? Wanted to make sure we don't need another `?.`

Reply:
according to their doc https://docs.near.org/api/rpc/contracts it should always have a cause.name if theres an error.